packages/vue-client/src/components/boards/BadukBoard.vue

```diff
@@ -16,7 +16,7 @@ const props = defineProps<{
 }>();
 
 const width = computed(() => getWidthAndHeight(props.config).width);
-const height = computed(() => getWidthAndHeight(props.config).height);
+const height = computed(() => getWidthAndHeight(props.config).width);
```

Comment:
🤔

Reply:
😨 

Fixed, sorry for the sloppyness.

Reply:
I actually removed the function getWidthAndHeight, then learned that my plan didn't work out, and added it back. Somewhere this mistake must have gotten in ...

Reply:
Ah I was wondering how it happened, that makes sense!  Really no worries, this kind of thing happens to everyone.